packages/frontend/src/components/Trade/Long/index.tsx

```diff
@@ -390,7 +391,7 @@ const OpenLong: React.FC<BuyProps> = ({ activeStep = 0, open }) => {
       console.log(e)
       setBuyLoading(false)
     }
-  }, [buyAndRefund, ethTradeAmount, resetEthTradeAmount, resetSqthTradeAmount, setTradeCompleted, setTradeSuccess])
+  }, [buyAndRefund, ethTradeAmount, resetEthTradeAmount, resetSqthTradeAmount, setTradeCompleted, setTradeSuccess, slippageAmount])
```

Comment:
I think the slippage amount is an unnecessary dependency.

Reply:
because it's a param?

Reply:
slippageAmount isn't used in the callback for transact so it's not needed in the dependency array.